src/main/resources/hudson/plugins/gradle/injection/init-script.gradle

```diff
@@ -14,6 +14,11 @@ initscript {
         return System.getProperty(name) ?: System.getenv(envVarName)
     }
 
+    // finish early if injection is disabled
+    if (getInputParam("GRADLE_ENTERPRISE_GRADLE_INJECTION_ENABLED") == null) {
```

Comment:
Env var name is not consistent with others as well as the call to the `getInputParam`. Maybe we should just change the initscript to only env var names are read ? It's easier to follow when looking for env var usage.

Reply:
I initially wanted it to be it's own separate thing and not bind it to Jenkins config as its not something user can influence. But I agree that it looks a bit odd in the init script (and in the code too for that matter) and we already had a similar example of `JENKINSGRADLEPLUGIN_GRADLE_AUTO_INJECTION`, so I renamed it to make the name consistent.